skit.API/Controllers/Areas/Auth/AccountController.cs

```diff
@@ -21,7 +21,7 @@ public sealed class AccountController : BaseController
     [HttpPost("sign-in")]
     [ProducesResponseType(StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
-    public async Task<ActionResult<JwtWebToken>> SignIn([FromBody] SignInCommand command, CancellationToken cancellationToken)
+    public async Task<ActionResult<JsonWebToken>> SignIn([FromBody] SignInCommand command, CancellationToken cancellationToken)
```

Comment:
Po prostu zmiana nazewnictwa?

Reply:
Bo JTW to skrót od JsonWebToken więc tą starą nazwa sensu nie miała